content/blog/posts/visual-studio-2022-update.md

```diff
@@ -0,0 +1,277 @@
+
+## Background
+
+When we say that **O3DE** "fully supports" Visual Studio 2022, we are referring to building O3DE projects using Visual Studio 2022 [v143 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset).  
+![Visual Studio 2022 v143 toolset](/images/blog/vs2022/vs2022-installer-v143-toolset.png)
+
+Because O3DE uses [CMake](https://www.o3de.org/docs/user-guide/build/configure-and-build/) as its build system generator, it has been possible since the release of [CMake 3.21](https://cmake.org/cmake/help/v3.21/release/3.21.html#generators) to configure and *attempt* to build using the Visual Studio 2022.  By *attempt*, we mean that building O3DE using Visual Studio 2022 could occur, but would have most likely failed due to stricter compiler warnings which were triggering using the newer v143 toolset.  
+The backend build server infrastructure for validating new commits to O3DE via the Automated Review(AR) pipeline wasn't setup to build against the v143 toolset.  
+Before the 22.10.0 release, continuous integration builds on Windows only installed the Visual Studio 2019(VS2019) Build Tools. The [v142 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset) was the only Microsoft Visual C++(MSVC) toolset validated at the time and therefore build breakages of O3DE building against the Visual Studio 2022 v143 toolset were common and took days to discover.
+
+## What's new
+
+### Build Server Updates
+Between the time of the 22.05.0(May 2022) release the 22.10.0(October 2022) release the Jenkins build servers have been updated with Visual Studio 2022 17.3 Build Tools and the latest v143 toolset. 
+Furthermore all O3DE build warnings and errors using the the v143 toolset were fixed and future VS2022 compile failures are protected against via the Automated Review pipeline validating each Pull Request that goes into O3DE repo [development](https://github.com/o3de/o3de) branch.  
+
+
+### Project Manager Updates
+The 22.10.0 release also included a feature update to the [Project Manager](https://www.o3de.org/docs/user-guide/project-config/project-manager/#projects) application. The requirement building against the Visual Studio 2019 application has been removed.  
+The Project Manager application will use the build toolset of latest version of Visual Studio installed on the user's machine for configuring and building O3DE Projects.  If the user only has Visual Studio 2019 installed with the v142 toolset, that will be used for compiling O3DE. If the user only has Visual Studio 2022 installed with the v143 toolset that will be used for compilation. Finally if the user has both Visual Studio 2019 and 2022 installed then Visual Studio 2022 v143 toolset will be used.
+![Project Manager Project Options](/images/blog/vs2022/project-manager-build-options.png)
```

Comment:
`![Project Manager Project Options](/images/blog/vs2022/project-manager-build-options.png)`
Change to
`{{< image-width src="/images/blog/vs2022/project-manager-build-options.png" width="100%" alt="Project Manager Project Options" >}}`